### PR TITLE
Update Bomber Guys to v1.0.3

### PR DESCRIPTION
**Pull request by Pe241**
### Changed

- Amount of TNT in kit (32→16)

- Removed jump restriction

- Remove tips repetition

- mode-change
 The central pillar disappears when the anvil falls.

- Moved some anvils

- Added anvils little more

### CODE REVIEW

File: Bomber Guys/map.xml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <map proto="1.4.0">
   <name>Bomber Guys</name>
-  <version>1.0.2</version>
+  <version>1.0.3</version>
   <objective>Blow up your enemy!</objective>
   <rules>
-    <rule>Jumping, Fall damage, Explosion damage by self and ally are disabled.</rule>
+    <rule>Fall damage, Explosion damage by self and ally are disabled.</rule>
     <rule>Don't try to hit the EGG Grenade on the Falling Anvil!</rule>
```
Comment (KingOfSquares) [inline Bomber Guys/map.xml:8]:
```xml
<rule>Fall damage is disabled.</rule>
<rule>You can not damage yourself or your team with TnT</rule>
<rule>Don't throw EGG Grenades on the Falling Anvil!</rule>
```

I think the sentences here need to be rephrased somehow, left a suggestion above, but unsure myself. Not even sure if its necessary to mention fall damage/TnT damage as a rule, but maybe a tip at the start? I think its pretty self explanatory when the map runs :)

File: Bomber Guys/map.xml [hunk 2]
```diff
@@ -138,7 +138,7 @@
   
     <apply kit="play" region="playable"/>
     <apply leave="never" region="playable" message="Don't go outside!"/>
-    <apply block="noanvil" region="outside" message="You can't edit outside!"/>
+    <apply block="noanvil" region="outside" message="You can only break a melon block!"/>
```
Comment (KingOfSquares) [inline Bomber Guys/map.xml:141]:
```xml
message="You can only break melon blocks!"
```

File: Bomber Guys/map.xml [hunk 3]
```diff
@@ -153,9 +153,9 @@
   </itemremove>
   <time>5m</time>
   <broadcasts>
-    <tip after="1s" every="1m">Melon Block is breakable! Let's blow up!</tip>
+    <tip after="1s">Melon Block is breakable! Let's blow up!</tip>
     <tip after="2s">After two and a half minutes, anvils will falling down on the `8`oblack stained glass `b`ofloor! Be careful!</tip>
-    <tip after="30s" every="1m">Hey guys! Did you notice that you can reduce the explosion damage by hiding behind a pillar? Try it!</tip>
+    <tip after="30s">Hey guys! Did you notice that you can reduce the explosion damage by hiding behind a pillar? Try it!</tip>
     <alert after="140s">`6Anvils fall down after 10 seconds! Stay away from the `8black stained glass `6floor immediately!</alert>
```
Comment (KingOfSquares) [inline Bomber Guys/map.xml:159]: `Melon Blocks are breakable! Blow them up!`
`Did you notice that you can reduce explosion damage by hiding behind a pillar?`
``` `6Anvils fall down after 10 seconds. Stay away from the `8black stained glass `6floor immediately! ```

**Review by KingOfSquares — COMMENTED** [review]
Left some comments about the tips/rules. Not necessary to use but I think some of the sentences could use a little rephrasing/grammar fixing

**Comment by Pe241** [comment]
Thank you @KingOfSquares !
I'll fix and re-request later. (Deleted base repository :P)

**Pull request closed without merging**